projects/storyteller/webapp/src/app/app.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="!storylist">
+    Loading
+</div>
+<div *ngIf="storylist">
+    <div *ngFor="let storylistChild of storylist">
```

Comment:
Usually each project has one intent size for all files.
Intent size means how much spaces is contained in one tab.

It's set here:
https://github.com/hasadna/hasadna/blob/finishing/projects/storyteller/webapp/.editorconfig#L7

The project has 2 spaces intent size. But the file has 4 per tab.
https://i.imgur.com/s6afFx2.jpg

Reply:
I will change my visual studio settings right now